lib/factory_bot/strategy/stub.rb

```diff
@@ -71,7 +71,7 @@ def destroyed?
         created_at_missing_default = result_instance.respond_to?(:created_at) && !result_instance.created_at
         result_instance_missing_created_at = !result_instance.respond_to?(:created_at)
 
-        if created_at_missing_default || result_instance_missing_created_at
+        if created_at_missing_default
```

Comment:
Because we're removing `result_instance_missing_created_at` (and `updated_at`, below), we should also remove the assignment of these values (on lines 72 and 84)